src/main/java/algorithms/curated170/medium/PlusOneLinkedList.java

```diff
@@ -0,0 +1,39 @@
+package algorithms.curated170.medium;
+ 
+import algorithms.datastructures.ListNode;
+
+public class PlusOneLinkedList {
+
+    public ListNode addOne(ListNode head)
```

Comment:
Can we change the method name to "plusOne" in order to be compatible with LeetCode?

Reply:
Quite a smart solution using sentinel node as a dummy head.
![image](https://user-images.githubusercontent.com/60903744/117544366-d0e8d680-b029-11eb-8a6a-c5317b8fe317.png)